src/EnergyPlus/HybridModel.cc

```diff
@@ -229,7 +464,7 @@ namespace HybridModel {
     void clear_state()
     {
 
-        FlagHybridModel = false;
+        FlagHybridModel_TM = false;
```

Comment:
Does any of other state need to be cleared, like FlagHybridModel_AI?

Reply:
For unit test purpose, all changed vars need to be re-initialized. All allocated arrays need to be deallocated.